litho-widget/src/main/java/com/facebook/litho/widget/TextInputSpec.java

```diff
@@ -156,6 +156,8 @@
  * @prop editable If set, allows the text to be editable.
  * @prop inputType Type of data being placed in a text field, used to help an input method decide
  *     how to let the user enter text. To add multiline use multiline(true) method.
+ * @prop rawInputType Type of data being placed in a text field, used to help an input method decide
```

Comment:
This doesn't dd a lot of context in terms how it is different from `inputType`. Can you add more specific guidance for users to pick one or another, maybe even references to Android docs.

Reply:
I have added the specific description from the Android docs - hope this is more clear now, but please let me know if there are further changes to make.